### PR TITLE
formatting buttons in edit project text editor

### PR DESCRIPTION
**Pull request by aasis21**
#3629 
Now, the formatting buttons now show its corresponding rst on the editor.

### CODE REVIEW

**Comment by davidfischer** [comment]
Firstly, thank you very much for the first-time contribution!

This does in fact fix the RST editor and it worked for me. However, I had a question about the minified file `jquery.markitup.pack.js`. Did you generate this file? I don't see it when I download the editor from http://markitup.jaysalvat.com. I'm hesitant to include minified JavaScript until I can verify the source.

Ideally, this plugin should come from bower but perhaps it's best to merge this PR once we've verified the pack file and then switch to loading it from bower in a separate PR.

**Comment by davidfischer** [comment]
I'm even starting to wonder whether we need this description field at all. It only shows up on project pages and isn't really used to do anything.

**Comment by aasis21** [comment]
Thanks for the review.

Regarding pack file, I did it online using the actual jquery.markitup.js file.

**Comment by aasis21** [comment]
Yes, the significance of description field is a another design question.

**Comment by ericholscher** [comment]
I'd be +1 on removing it. It's a bunch of code for a small field, which most people will just be copying from a README file or similar.

Sorry to invalidate the work you did @aasis21, but if you could just remove the files, I'd be happy to commit it! Thanks for making the effort, allowing us to have this discussion.

**Comment by aasis21** [comment]
No problems @ericholscher, I'd be happy to do that. So, what you are saying is to completely remove the description feature from the project. Am I right?

**Comment by agjohnson** [comment]
I think this PR should be scaled back to just remove the markitup dependency. We can keep a description field for now. Removing this will be a larger chunk of work and requires us to make a design decision around this. We'll want to review the UI changes that this entails for before just removing.

I've have been wanting to get rid of this field for a long time however. I don't think we need it, or any project description on our dashboard page -- this isn't a page that readers see.

File: readthedocs/templates/projects/project_create.html [hunk 1]
```diff
@@ -1,23 +1,6 @@
 {% extends "dashboard/base_dashboard.html" %}
 {% load i18n %}
 
-{% block extra_links %}
-  <link rel="stylesheet" type="text/css" href="{{ MEDIA_URL }}lib/markitup/skins/simple/style.css" />
-  <link rel="stylesheet" type="text/css" href="{{ MEDIA_URL }}lib/markitup/sets/sphinx/editor.css" />
-{% endblock %}
-
-{% block dash-nav-create %}active{% endblock %}
```
Comment (agjohnson) [inline readthedocs/templates/projects/project_create.html]: This is a required line.

Comment (aasis21) [reply readthedocs/templates/projects/project_create.html]: Sorry, it was removed by mistake. Thanks for pointing out.

**Comment by aasis21** [comment]
Thanks, @agjohnson for clarifying the situation. I have made changes as you suggested.

**Comment by ericholscher** [comment]
Thanks so much for this PR @aasis21! I love PR's that delete code, and this one deletes a lot :)

**Comment by aasis21** [comment]
Thanks @ericholscher

**Pull request merged**